### ISSUE TITLE
A11y - Loading page not announced

### ISSUE THREAD

**Issue by sv-ohorvath** [mozilla-mobile/focus-android#4121]
## Steps to reproduce
1. Turn on TalkBack screen reader.
2. Load a page in Focus.
### Expected behavior
Talkback should read "loading <webpage url>" (#2783)
### Actual behavior
When a page starts loading it is not announced, just when the page is fully loaded. 
### Device information

* Android device: Pixel (Android 9)
* Focus version: Nightly & Beta 8.0.5 Gv. Not reproducing on WV.

**Comment by BinduVerma123** [comment]
@sv-ohorvath can i work on this issue?

**Comment by sv-ohorvath** [comment]
Hi @BinduVerma123 ! I'm not the right person to answer that. @vesta0 maybe can help.

**Comment by BinduVerma123** [comment]
@vesta0 can i work on this issue?

**Comment by sflorean** [comment]
Reproducible on Focus: 8.3.0 (Build ID: 341281936) with Samsung Galaxy Note10 (Android 10).

**Comment by lobontiumira** [comment]
This is still reproducible on 8.16.0 Focus build with Samsung Galaxy Note 8 (Android 9).

**Comment by lobontiumira** [comment]
This issue is no longer reproducible on Focus Nightly 97.0 from 1/3.
After double-tapping on "go", the talckback announces that the page is loading.
Tested with Oppo Find X3 Lite (Android 11).
